tests/pytest/conftest.py

```diff
@@ -104,6 +104,14 @@ def model_LittlepayGroup(model_EnrollmentFlow):
     )
 
 
+@pytest.fixture
+def model_SwitchioGroup(model_EnrollmentFlow):
+    return SwitchioGroup.objects.create(
+        group_id="veteran-discount",
```

Comment:
Not a big deal since these are all mocks, I just got a little confused for a second while running through a test since the `model_EnrollmentFlow` fixture corresponds to `senior`. Maybe we can call this `senior-discount` instead? We'd have to update the `group` name in the `mocked_group_expiry_no_expiry` and `mocked_group_expiry_with_expiry` fixtures too.